app/assets/stylesheets/vars/_utils.scss

```diff
@@ -0,0 +1,37 @@
+// Border radius
+$br-default: 3px;
+
+// Break points
+$breakpoint-s: 640px;
+$breakpoint-m: 768px;
+$breakpoint-l: 1024px;
+$breakpoint-xl: 1280px;
+
+// Transitions
+$transition-props: cubic-bezier(0.17, 0.67, 0.5, 0.71) 100ms;
+
+// pow() - Raise number to the nth power
+//
+// @param {number} $base      The base number
+// @param {number} $exponents The exponent to which to raise $base
+@function pow($base, $exponents) {
+  $raised: 1;
+
+  @for $i from 1 through $exponents {
+    $raised: $raised * $base;
+  }
+
+  @return $raised;
+}
+
+// Generating classes for Tailwind approach.
+@mixin generate-classes($prefix, $properties, $values1, $values2: ()) {
```

Comment:
Having `values1` and `values2` as separate parameters helps with situations when I wanna use a list of existing values and [combine them with other values](https://github.com/thepracticaldev/dev.to/pull/6334/files#diff-5f06803a04b2e9c226669b9fb6063bedR4-R13).